person.rb

```diff
@@ -1,20 +1,31 @@
-class Person
+require './nameable'
+require './rental'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rental
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- [ ] Similar to the book case, a person can have many rentals but the function `add_rental` to this attribute is missing, please add it. I'd suggest taking a look at [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).